CONTRIBUTING.md

```diff
@@ -4,5 +4,5 @@ PyBee <3's contributions!
 
 Please be aware, PyBee operates under a Code of Conduct. 
 
-See [CONTRIBUTING to PyBee](pybee.org/contributing) for details.
+See [CONTRIBUTING to PyBee](pybee.org/contributing/index.html) for details.
```

Comment:
Well spotted - but I think the problem is the missing `http://`, not the `index.html` at the end.